### DISCUSSION TITLE
Full message-level encryption support

### DISCUSSION THREAD

**Discussion by tpimh** [OpenIDC/mod_auth_openidc#870] [General]
I'm now setting up mod_auth_openidc and [Full message-level encryption](https://developer.signicat.com/enterprise/identity-methods/ftn/#full-message-level-encryption) is a requirement. Is it possible to implement MLE with mod_auth_openidc?

**Comment by zandbelt** [comment]
Yes, that is possible by configuring [`OIDCPublicKeyFiles`](https://github.com/zmartzone/mod_auth_openidc/blob/v2.4.11.2/auth_openidc.conf#L182-L190) and [`OIDCPrivateKeyFiles`](https://github.com/zmartzone/mod_auth_openidc/blob/v2.4.11.2/auth_openidc.conf#L192-L197) and send the public cert to the OP.
